modules/thunder_liveblog/thunder_liveblog.install

```diff
@@ -148,3 +150,28 @@ function thunder_liveblog_install() {
   }
 
 }
+
+/**
+ * Adds image to the list of available paragraphs.
+ */
+function thunder_liveblog_update_8101() {
+  /** @var \Drupal\thunder_updater\Updater $thunderUpdater */
+  $thunderUpdater = \Drupal::service('thunder_updater');
+
+  /** @var \Drupal\field\Entity\FieldConfig $field */
+  $field = FieldConfig::load('liveblog_post.liveblog_post.field_embed_media');
+
+  if ($field) {
```

Comment:
BTW, this is also true for other tests, e.g. in the `thunder.install` file. Maybe we should adjust them.